### PR TITLE
bug/2769_invalid_regular_expression_for_idPattern

### PR DESCRIPTION
**Pull request by kzangeli**
Just a functest to prove that invalid regex in idPattern does not break the broker (for issue #2769).

### CODE REVIEW

File: test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test]: It seems this exact case is already covered by cases/2257_crash_with_invalid_subscription_when_matching/crash_with_invalid_subscription_when_matching.test (wihouth step 2 in that case).

Comment (kzangeli) [inline test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test]: So, let's just throw the PR away then.
Right?

Comment (fgalan) [inline test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test]: Two alternatives:

* Throw the PR away
* Throw the invalid_regular_expression_for_idPattern.test away, after moving its step 2 to crash_with_invalid_subscription_when_matching.test

Comment (kzangeli) [inline test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test]: ok, I'll add the "make sure the sub wasn't created" to that other functest and delete the new functest.

Comment (kzangeli) [inline test/functionalTest/cases/2769_invalid_regular_expression_for_idPattern/invalid_regular_expression_for_idPattern.test]: Fixed in 448fb23

**Comment by fgalan** [comment]
LGTM

**Pull request merged**